krustlet-wascc-e2e.stderr.txt

```diff
@@ -0,0 +1,45 @@
+[2020-09-23T20:10:21Z INFO  wascc_host::bus::inproc] Initialized Message Bus (internal)
```

Comment:
This seems like it should be added to `.gitignore`.

Reply:
Yeah, we should add these output files to gitignore for sure

Reply:
Yep, sorry about this!